README.md

```diff
@@ -126,5 +127,23 @@ dev:
     multi-az: false
     db-engine: postgres
     db-engine-version: 9.3.5
+  ssl:
+    my-cert:
+      cert: |
+        -----BEGIN CERTIFICATE-----
+        blahblahblah
+        -----END CERTIFICATE-----
+      key: |
+        -----BEGIN RSA PRIVATE KEY-----
+        blahblahblah
+        -----END RSA PRIVATE KEY-----
+      chain: |
+        -----BEGIN CERTIFICATE-----
+        blahblahblah
+        -----END CERTIFICATE-----
 ```
 
+##### SSL certs for ELBs
+If you set the protocol on an ELB to HTTPS you must include a key called `certificate_name` and matching cert data in a key with the same name under `ssl` (see example above). The `cert` and `key` are required and the `chain` is optional.
```

Comment:
`certificate_name`  given the example YAML shuld be `my-cert` I believe.

We also perhaps should make it clear that we recommend storing the cert in the secrets/passwords YAML, not the main project one.